h/api/storage.py

```diff
@@ -44,6 +46,154 @@ def fetch_annotation(request, id):
     return elastic.Annotation.fetch(id)
 
 
+def _legacy_create_annotation_in_elasticsearch(request, data):
+    annotation = elastic.Annotation(data)
+    # FIXME: this should happen when indexing, not storing.
+    _prepare(request, annotation)
+    annotation.save()
+    return annotation
+
+
+def transform_links(document, claimant, created, updated):
```

Comment:
The one argument I can think of for passing in `created` and `updated` (rather than letting the db auto generate them) is that each `DocumentURI` will have the same `created` value as the annotation that created it, and the same `updated` value as the annotation that last updated. Also the `DocumentURI`'s `created` and `updated` will be exactly the same as each other, for ones that have never been updated. Rather than all of these times being milliseconds different to each other. Whether that's actually important at all I'm not sure but it does mean that comparisons like `created == updated` will work, if we ever need them.

I noticed that with the `Timestamps` mixin we're using, a new annotation gets `created` and `updated` times that are actually slightly different to each other.